packages/weak-memoize/types/index.d.ts

```diff
@@ -1,4 +1,6 @@
-type UnaryFn<Arg, Return> = (arg: Arg) => Return
+// TypeScript Version: 2.2
+
+export type UnaryFn<Arg, Return> = (arg: Arg) => Return
```

Comment:
similarly as at another place - does it have to be exported?

Reply:
Added the export because the linting rules were failing. Will remove the export and disable that linting rule